local/setup.sh

```diff
@@ -14,7 +14,7 @@ MINIO_SECRET_KEY="secret_key"
 MINIO_BUCKETS=( "static" "artifacts" )
 
 # Quit immediately when docker-compose receives a Ctrl+C
-trap SIGTERM exit
+trap exit EXIT
```

Comment:
My reading of the documentation I found in man bash does not indicate that EXIT is appropriate here, I would've expected SIGTERM (still in the "exit SIGTERM" order though).

Maybe I'm missing something though?

> If  a  sigspec  is  EXIT  (0)  the command arg is executed on exit from the shell.

Reply:
Hmm, this is weird. If I keep `trap exit EXIT` the container will stop as soon as I hit `ctrl+c` in the docker-compose terminal, while if I replace it with `trap exit SIGTERM` (or `trap exit SIGINT`) it will instead hang. I can confirm your quote is what the docs say....

Reply:
Huh. Well, seems fine, though maybe we should add a note here.